addons/account/models/chart_template.py

```diff
@@ -335,6 +335,8 @@ def tax_template_changed(tax, template):
                             }])
                             account = existing_account
 
+                    if 'reconcile' in values:
+                        del values['reconcile']
```

Comment:
Could you add a comment mentioning the reason? (avoid to override user custom settings & partial reconcile error) :)

Reply:
thanks for the review @JulienAlardot 
I have applied changes. commit message also changed.

there is no any ticket as this was generated in upgrade request during upgrade.